core/tests/puppeteer-acceptance-tests/user-utilities/exploration-editor-utils.ts

```diff
@@ -169,14 +183,14 @@ export class ExplorationEditor extends BaseUser {
    * @param {string} answer - The response to be added.
    * @param {string} feedback - The feedback for the response.
    * @param {string} destination - The destination state for the response.
-   * @param {string} correctness - Whether the response is marked as correct.
+   * @param {boolean} correctness - Whether the response is marked as correct.
    */
   async addResponsesToTheInteraction(
```

Comment:
Aren't you adding just one response? If so then why is the function name plural?

Reply:
oh, certainly, the name shouldn't be plural. 
Changed to  addResponseToTheInteraction().